hazelcast-spring/src/main/resources/hazelcast-spring-3.9.xsd

```diff
@@ -1471,7 +1471,7 @@
         <xs:complexType>
             <xs:complexContent>
                 <xs:extension base="hazelcast-bean">
-                    <xs:all>
+                    <xs:choice minOccurs="1" maxOccurs="unbounded">
```

Comment:
I had to look up that XSD stuff, since I don't know it by heart:
* `<xs:all>` means that all elements can appear in any order, but all elements MUST occur once and only once (which prevents multiple Near Caches)
https://www.w3schools.com/xml/el_all.asp
* `<xs:choice minOccurs="1" maxOccurs="unbounded">` means that all elements can appear in any order, but all elements MUST occur once (and can be there unlimited time, which allows multiple Near Caches)
https://www.w3schools.com/xml/el_choice.asp

What I don't understand is the `minOccurs="1"`. What if I don't have a Near Cache? This sounds wrong to me.